packages/components/src/components/MultiDistributionChart/index.tsx

```diff
@@ -108,39 +116,36 @@ const InnerMultiDistributionChart: FC<{
       const getColor = (i: number) =>
         plot.colorScheme === "blues" ? "#5ba3cf" : d3.schemeCategory10[i];
 
-      const {
-        padding,
-        chartWidth,
-        chartHeight,
-        xScale,
-        yScale,
-        translateToZero,
-      } = drawAxes({
+      const xScale = scaleTypeToScale(settings.xScale);
+      xScale.domain([
+        Number.isFinite(settings.minX)
+          ? settings.minX!
+          : d3.min(domain, (d) => d.x) ?? 0,
+        Number.isFinite(settings.maxX)
+          ? settings.maxX!
+          : d3.max(domain, (d) => d.x) ?? 0,
```

Comment:
Default max is 0? I'm sure there's a reason, but kind of caught me by surprise

Reply:
This code is not new (see the removed lines below). It doesn't really matter, `0` only happens if the dist has any points, and then there's nothing to draw.